packages/combo-box/src/vaadin-combo-box-mixin.js

```diff
@@ -921,14 +921,11 @@ export const ComboBoxMixin = (subclass) =>
       }
 
       if (this._isValidValue(value)) {
-        let item;
```

Comment:
There was a conflict here, due to `isValidValue` extracted into a function on master.